### PR TITLE
bumping `winit` to 0.29.3

### PR DESCRIPTION
**Pull request by TornaxO7**
**Checklist**

- [x] Run `cargo clippy`.
- [x] Run `cargo clippy --target wasm32-unknown-unknown` if applicable.
- [x] Add change to CHANGELOG.md. See simple instructions inside file.

**Connections**
None

**Description**
Just bumping `winit` to from `0.29.2` to `0.29.3`

**Testing**
Eh

### CODE REVIEW

File: Cargo.toml [hunk 1]
```diff
@@ -123,7 +123,7 @@ wgpu-example = { version = "0.18.0", path = "./examples/common" }
 wgpu-macros = { version = "0.18.0", path = "./wgpu-macros" }
 wgpu-test = { version = "0.18.0", path = "./tests" }
 wgpu-types = { version = "0.18.0", path = "./wgpu-types" }
-winit = { version = "0.29.2", features = [ "android-native-activity" ] }
+winit = { version = "0.29.3", features = [ "android-native-activity" ] }
```
Comment (cwfitzgerald) [inline Cargo.toml]: We can leave this at 0.29 - and just update the lock file.

**Review by cwfitzgerald — APPROVED** [review]
Nit, but otherwise I'm the future dependabot does this for us, so we don't need to do it manually

File: Cargo.toml (hunk 1 above)
Comment (TornaxO7) [reply Cargo.toml]: so, you mean, I can just remove the third number (here the `3`)?

Comment (cwfitzgerald) [reply Cargo.toml]: Yes - it's equivalent to writing 0.29.0

Comment (TornaxO7) [reply Cargo.toml]: Done!

**Pull request merged**